docs/detections/rules-cross-cluster-search.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[rules-cross-cluster-search]]
+== Cross-cluster search and detection rules
+
+{ref}/modules-cross-cluster-search.html[Cross-cluster search] is an {es} feature that allows one cluster (the _local_ cluster) to query data in a separate cluster (the _remote_ cluster). {elastic-sec}'s detection rules can perform cross-cluster search to query data in remote clusters.
+
+[discrete]
+[[set-up-ccs-rules]]
+=== Set up cross-cluster search in detection rules
+
+This section explains the general process for setting up cross-cluster search in detection rules. For specific instructions on each part of the process, refer to the linked documentation.
+
+. On the local cluster, {ref}/remote-clusters-connect.html[set up a connection] to the remote cluster. Note the unique name that you give to the remote cluster, because you'll need to use it throughout this process.
+
+. On both the local and remote clusters, {ref}/remote-clusters-privileges.html#clusters-privileges-ccs-kibana[create a role for cross-cluster search privileges], and make sure the two roles have _identical_ names. Assign each role the following privileges:
+
+.. *Local cluster role*: Assign the `read` privilege to the indices you want to search, using _both_ the local and remote index patterns for each index. To specify a remote index, use the pattern `<remote_cluster_name>:<index_name>`.
++
+For example, if the remote cluster's name is `remote-security-data` and you want to query the `logs-*` indices, include both `logs-*` and `remote-security-data:logs-*` and assign them the `read` privilege.
```

Comment:
Should this be index patterns instead of indices? 

```suggestion
For example, if the remote cluster's name is `remote-security-data` and you want to query the `logs-*` index pattern, include both `logs-*` and `remote-security-data:logs-*` and assign them the `read` privilege.
```

I feel like both might work in this case, but one might be more technically correct than the other. I also hesitate to suggest this change because I understand that you're saying "if you want to query all of the log indices" or "all of the indices that begin with `logs-`" -- so basically it works as is. Wdyt?